Vinyl/Turntable.swift

```diff
@@ -32,31 +34,55 @@ public final class Turntable: NSURLSession {
             operationQueue = NSOperationQueue()
             operationQueue.maxConcurrentOperationCount = 1
         }
+        
+        if configuration.recodingEnabled {
+            recorder = Recorder(wax: Wax(tracks: []), recordingPath: configuration.recordingPath)
+            recordingSession = urlSession ?? NSURLSession.sharedSession()
+        }
```

Comment:
`recordingSession` is optional because:

1.  We don't always need one. Seems silly to get reference to `NSURLSession.shareSession()` when you're not recording just to have a constant.
2.  `recordingSession` being `nil` or not is used to determine whether any recording needs to be persisted in [`stopRecording()`](https://github.com/Velhotes/Vinyl/pull/64/files#diff-6a97ae6da4476a6cd6b34c3c369de7daR74)